omnigibson/prims/rigid_prim.py

```diff
@@ -201,7 +201,7 @@ def update_meshes(self):
                     # If we're not a valid volume, use bounding box approximation for the underlying collision approximation
                     if not is_volume:
                         log.warning(f"Got invalid (non-volume) collision mesh: {mesh.name}")
-                        mesh.set_collision_approximation("boundingCube")
+                        # mesh.set_collision_approximation("boundingCube")
```

Comment:
is this no longer needed?

Reply:
Yes, per discussion with @cgokmen . I think our conclusion was that we should never do this anymore, or rather that (a) all of our assets should never have this problem, and (b) any edge cases we encounter we should explicitly address